Program.cs

```diff
@@ -45,18 +47,17 @@ class Program {
          Console.OutputEncoding = Encoding.UTF8;
          for (int i = 1; i <= sFinalSoln.Count; i++) {
             Console.CursorTop = 2; Console.CursorLeft = 0;
-            Console.WriteLine ($"Solution {i} of {sFinalSoln.Count}");
-            var soln = sFinalSoln[i - 1];
-            Console.WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
-            for (int j = 0; j < 7; j++) {
+            Console.Write ($"Solution {(i == 0 ? i + 1 : i)} of {sFinalSoln.Count}");

```

Comment:
Simply adding an empty space after the Solution count would fix the issue.
```suggestion
            Console.Write ($"Solution {(i == 0 ? i + 1 : i)} of {sFinalSoln.Count} ");
```